src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -35,9 +32,10 @@ public class JsonOperationContext : IDisposable
 
         private int _lastStreamSize = 4096;
 
-        public JsonOperationContext(UnmanagedBuffersPool pool)
+        public JsonOperationContext()
         {
-            Pool = pool;
+            _arenaAllocator = new ArenaMemoryAllocator();
+            _arenaAllocatorForLongLivedValues = new ArenaMemoryAllocator(16 * 1024);
```

Comment:
a) ByteStringContext is an allocator based memory manager that returns an struct that points to chunks of memory. The fact we named it ByteString is just because we designed it originally to replace strings, but could have been called BytesSegment or MemoryChunk and would have been the same. What I am worried about is that keeping 2 sources of unmanaged pooled memory sources could be troublesome if not managed properly.
b) That makes much more sense, so we are pooling the entire context and as they become long lived fragmentation doesnt happen.